torchgeo/datasets/eurosat.py

```diff
@@ -174,6 +188,46 @@ def _extract(self) -> None:
         filepath = os.path.join(self.root, self.filename)
         extract_archive(filepath)
 
+    def plot(
+        self,
+        sample: Dict[str, Tensor],
+        show_titles: bool = True,
+        suptitle: Optional[str] = None,
+    ) -> plt.Figure:
+        """Plot a sample from the dataset.
+
+        Args:
+            sample: a sample returned by :meth:`__getitem__`
+            show_titles: flag indicating whether to show titles above each panel
+            suptitle: optional string to use as a suptitle
+
+        Returns:
+            a matplotlib Figure with the rendered sample
```

Comment:
Can you add:
```rst
.. versionadded:: 0.2
```
to any new functions/classes that get added? We've been forgetting to do this but I updated a few things in 85768b81732e763a0ec742ae0f7a2f7c5aa44000. If you see anything I missed please add it.

Reply:
yep!